### DISCUSSION TITLE
Global parameters yaml file

### DISCUSSION THREAD

**Discussion by Serega6678** [Lightning-AI/pytorch-lightning#7940] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I am using CLI to train my model. Instead of specifying parameters directly, I provide a yaml file with variables defined.

Since I'm using several loggers, they have a common name parameter. So in order to start a new experiment I have to change this parameter in each logger. This raises a question is there a way to create global variable in yaml file while using CLI?

**Comment by tshu-w** [comment, marked as answer]
@Serega6678 You can try jsonnet format config file which supports global variables. https://jsonargparse.readthedocs.io/en/stable/#jsonargparse.core.ArgumentParser
To use jsonnet format, you can init CLI by passing `{"parser_mode": "jsonnet"}` to `parser_kwargs`.

**Reply by Serega6678** [reply]
Thank you very much. Appreciate your time
